custom/covid/management/commands/update_all_activity_complete_date.py

```diff
@@ -51,19 +62,107 @@ def handle(self, domain, **options):
             )
 
 
-def _get_bad_case_ids(domain):
-    return list(CaseSearchES()
-                .domain(domain)
-                .case_property_query("all_activity_complete_date", "date(today())")
-                .modified_range(gte=datetime.date(2022, 2, 2))
-                .case_type(["patient", "contact"])
-                .scroll_ids())
+def _get_bad_case_ids(domain, case_type):
```

Comment:
Without the domain filter, this matches 1,430,800 patients and 1,792,743 contacts on prod

Reply:
That's a total of 3.22 million, which is about the same as last time, though I didn't keep track of an exact number there.  It'd probably be prudent to add better logging there.